app/cache/get-cache-value.js

```diff
@@ -0,0 +1,10 @@
+const getCacheValue = async (cache, key) => {
+  console.log(`Getting cache value for key: ${key}`)
+  try {
+    return await cache.get(key)
```

Comment:
Can remove await here.

Reply:
Can't remove this -- I also thought same as yerself becuz its in an `async` func but a las, I cannot